language/move-lang/tests/move_check/liveness/trailing_semi.move

```diff
@@ -0,0 +1,64 @@
+script {
+    fun main() {
+        loop {
```

Comment:
I don't see a testcase where the divergent expression is an infinite loop (with no break). We should have one of those, right?

Reply:
Good catch! I'll add one